controllers/api/showRoutes.js

```diff
@@ -0,0 +1,94 @@
+const router=require('express').Router();
+const sequelize = require('../../config/connection')
+const {Show} = require('../../models');
+
+
+// Find all shows 
+router.get('/',async(req,res)=>{
```

Comment:
Remove the async here. This didn't return a response for me until I removed the async